R/generate_variability_terms.R

```diff
@@ -131,8 +133,13 @@ generate_ruv <- function(
   if (!is.null(seed)) {
     set.seed(seed)
   }
-  ruv$prop <- 1 + rnorm(nrow(ruv), 0, prop)
-  ruv$add <- rnorm(nrow(ruv), 0, add)
+  if (!ltbs) {
+    ruv$prop <- 1 + rnorm(nrow(ruv), 0, prop)
+    ruv$add <- rnorm(nrow(ruv), 0, add)
+    } else {
+      ruv$prop <- 1 + rnorm(nrow(ruv), 0, add)
+      ruv$add <- 0
+    }
```

Comment:
We follow the [tidyverse styleguide](https://style.tidyverse.org/syntax.html?q=inde#indenting), so make sure indentations are correctly formatted. 

We should also add a code comment about how ltbs works, since one might expect to see an actual log transform somewhere.

I also think it is slightly clearer to read "if ltbs, do x, else do y" as opposed to "if not ltbs do x, else do y", but that's more of a matter of personal taste.

Combining all the above, I would change this code block to look something like this, feel free to adapt as you feel is appropriate. 

```suggestion
  if (ltbs) {
    # data is returned from `PKPDsim::sim` in the linear domain already so
    # apply additive error (log domain) as proportional error (linear domain).
    ruv$prop <- 1 + rnorm(nrow(ruv), 0, add)
    ruv$add <- 0  
  } else {
    ruv$prop <- 1 + rnorm(nrow(ruv), 0, prop)
    ruv$add <- rnorm(nrow(ruv), 0, add)
  }
```